src/customJs/tools/promo_code/index.test.ts

```diff
@@ -89,9 +89,9 @@ describe(sut.name, () => {
     const result = sut();
 
     // Assert
-    const propertyGroup = result!.options.promo_code;
+    const propertyGroup = result!.options.promo_code_store;
```

Comment:
Similarly here, I think it sounds better as `store_promo_code`, because `store` acts as an adjective, and the noun is `promo_code`.